src/web/services/feedback-responses.service.ts

```diff
@@ -122,6 +126,35 @@ export class FeedbackResponsesService {
     }
   }
 
+  /**
+   * Determines whether responses should be displayed based on the selected section.
+   */
+  isFeedbackResponsesDisplayedOnSection(section: string, sectionType: InstructorSessionResultSectionType,
```

Comment:
A more logic way of organizing parameters would be to put `response` as the first argument.